### PR TITLE
Add first-round drafts for k8s 1.12 Release Team

### PR DESCRIPTION
**Pull request by justaugustus**
ref: #167 

/assign @tpepper @jberkus @jdumars 
/cc @tpepper @AishSundar @kacole2

Signed-off-by: Stephen Augustus <foo@agst.us>

### CODE REVIEW

File: releases/release-1.12/release_team.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+| CI Signal || Christian Hernandez ([@christianh814](https://github.com/christianh814)) |
+| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)) |
 | Bug Triage |||
 | Branch Manager |||
-| Docs |||
+| Docs | Zach Arnold ([@zparnold](https://github.com/zparnold)) | Samuel Tauil ([@samueltauil](https://github.com/samueltauil)) |
 | Release Notes |||
-| Communications |||
+| Communications | Kaitlyn Barnard ([@kbarnard10](https://github.com/kbarnard10)) ||
 | Patch Release Manager |||
```
Comment (feiskyer) [inline releases/release-1.12/release_team.md]: I'd like to be the Patch Release Manager

**Comment by dims** [comment]
@jberkus +1 to `"PR Triage"` as a new position. Let's leave it empty for now, let's see if we get anyone else who wants to help with it first.

File: releases/release-1.12/release_team.md (hunk 1 above)
Comment (justaugustus) [reply releases/release-1.12/release_team.md]: Sounds good, @feiskyer. Thanks for volunteering!

**Comment by calebamiles** [comment]
If we're proposing adding more positions, @jberkus, I would suggest also adding a role focused on upgrade/downgrade testing

**Comment by justaugustus** [comment]
Pending filling all lead roles...
/hold

**Comment by tpepper** [comment]
Regarding adding PR Triage:  depending on how the future happens there's shifting overlaps between feature/PM oversight, issue triage, PR triage, and CI signal.  MASSIVE CI signal attention seems to have lead to notably less issues in 1.11.  The removal of requiring issues for PRs seems to have lead to less issues in 1.10 and 1.11 as well.  But both of these reducing issue counts and issue connection to the milestone means less artifacts visible to the release team.  In this context, while it incurs overhead on staffing, I like having these roles as separate.  More pairs of eyeballs across this set makes for a more diverse risk management stance for the overall release.

File: releases/release-1.12/release_team.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-| CI Signal |||
-| Test Infra |||
-| Bug Triage |||
-| Branch Manager |||
-| Docs |||
-| Release Notes |||
-| Communications |||
-| Patch Release Manager |||
+| Lead | Tim Pepper ([@tpepper](https://github.com/tpepper)) | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) |
+| Features | Stephen Augustus ([@justaugustus](https://github.com/justaugustus)) | Kendrick Coleman ([@kacole2](https://github.com/kacole2)), Robert Sandoval ([@robertsandoval](https://github.com/robertsandoval)), Rajendar Talatam ([@rajendar38](https://github.com/rajendar38)) |
```
Comment (idvoretskyi) [inline releases/release-1.12/release_team.md]: Is it fine that we'll have multiple shadows for this role?

Not against it, just would like to clarify.

File: releases/release-1.12/release_team.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+| CI Signal || Christian Hernandez ([@christianh814](https://github.com/christianh814)) |
+| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)), Paul Renato ([@prwitt](https://github.com/prwitt)) |
```
Comment (idvoretskyi) [inline releases/release-1.12/release_team.md]: The same question (Is it fine that we'll have multiple shadows for this role?)

File: releases/release-1.12/release_team.md (hunk 2 above)
Comment (justaugustus) [reply releases/release-1.12/release_team.md]: Yes, we're trying to overstaff this release cycle as:
1. We're doing it much earlier than usual, so if prior commitments prevent people from participating, we will still be reasonably staffed.
2. More eyes on every role should provide better coverage and allow us to iterate faster on improvements.

**Comment by justaugustus** [comment]
@tpepper @jberkus -- we've got most of the roles staffed with multiple shadows at this point.
Can y'all drop an `/lgtm` on this?

**Comment by tpepper** [comment]
I'm trying to track down a branch manager, but this may as well merge in the meantime for visibility.

/lgtm

**Comment by tpepper** [comment]
Also for the record, we discussed back and forth and are going to continue with "Bug Triage" which will cover both GitHub issue and PR type artifacts.

**Comment by jberkus** [comment]
Let's see if I have approve powers:

/approve
/lgtm

**Pull request merged**